pixart.js

```diff
@@ -1 +1,35 @@
+var colorField;
+var chosenColor;
 
+var setBoxColor = function(event){
+  event.preventDefault();  //if don't have this page is refreshed
+  colorField = document.getElementById("color-field")
+  chosenColor = colorField.value
+  var brush = document.querySelector(".brush")
```

Comment:
i think you could just set this instantiation on lines 1 and 2 when they're being declared, as well as the brush